src/libraries/sendtohelixhelp.proj

```diff
@@ -210,6 +210,22 @@
       <SeleniumUrl>https://storage.googleapis.com/chromium-browser-snapshots/Linux_x64/$(ChromiumRevision)/chromedriver_linux64.zip</SeleniumUrl>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(RuntimeFlavor)' == 'CoreCLR' and '$(BUILD_BUILDID)' != ''">
+      <HelixPostCommands Condition="'$(TargetsWindows)' == 'true'">
+        $(HelixPostCommands);
+        %HELIX_PYTHONPATH% %HELIX_CORRELATION_PAYLOAD%\gen-debug-dump-docs.py -buildid $(BUILD_ID) -workitem %HELIX_WORKITEM_ID% -jobid %HELIX_CORRELATION_ID% -outdir %HELIX_WORKITEM_UPLOAD_ROOT%
+      </HelixPostCommands>
+      <HelixPostCommands Condition="'$(TargetsWindows)' != 'true'">
```

Comment:
One very big caveat here is macOS. It's only supported to debug them in dotnet-dump if it's a `createdump` dump. If it's a system dump, then only lldb works.

Reply:
I see. I'll mention it in the doc at the end.